src/dataproxy/worker/shared-worker.ts

```diff
@@ -33,6 +39,8 @@ const dataProxy: DataProxyWorker = {
       doctypes: [FILES_DOCTYPE, CONTACTS_DOCTYPE, APPS_DOCTYPE],
       initialSync: true,
       periodicSync: false,
+      syncDebounceDelayInMs: REPLICATION_DEBOUNCE,
+      syncDebounceMaxDelayInMs: REPLICATION_DEBOUNCE_MAX_DELAY,
```

Comment:
https://github.com/cozy/cozy-client/pull/1553 must be merged first and new version included in this commit